src/Tools/ResponseStrategies/ResponseFileStrategy.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace Mpociot\ApiDoc\Tools\ResponseStrategies;
+
+use Illuminate\Routing\Route;
+use Mpociot\Reflection\DocBlock\Tag;
+
+/**
+ * Get a response from from a file in the docblock ( @responseFile ).
+ */
+class ResponseFileStrategy
+{
+    public function __invoke(Route $route, array $tags, array $routeProps)
+    {
+        return $this->getFileResponse($tags);
+    }
+
+    /**
+     * Get the response from the file if available.
+     *
+     * @param array $tags
+     *
+     * @return mixed
+     */
+    protected function getFileResponse(array $tags)
+    {
+        $responseTags = array_filter($tags, function ($tag) {
+            return $tag instanceof Tag && strtolower($tag->getName()) == 'responsefile';
+        });
+        if (empty($responseTags)) {
+            return;
+        }
+        $responseTag = array_first($responseTags);
+
+        $json = json_decode(file_get_contents(storage_path($responseTag->getContent()), true), true);
```

Comment:
I'm wondering if it's really a good idea to use `storage_path` here. Some projects might use external storage such as S3, but would probably keep these response files in the repository. Maybe we make this a config option, similar to that of `response_calls`.

Reply:
What if I use **Storage::get('somefile')**, to get the default configuration from config/filesystems.php? Or do you think is better to have an option in config/apidoc.php?

Reply:
Please, check my last commit

Reply:
Yes, I understood. I was wondering if we would override the configuration by putting the driver filesystem in the config/apidoc.php file. But I already used the default filesystem driver, check the last commit. =)